SU-UComm/stanford-report
# Fix/carousels

# READY FOR REVIEW

# Summary
- Removing focus when mouse navigation is used.

# Criticality
### The Component(s) that need to be deployed:
- stories-carousel
- media-carousel
- image-gallery-modal
- content-carousel
- basic-story-hero

### Packages changed: packages/Carousel/Carousel.jsx

# Review Tasks

### Test links
- https://sug-web.matrix.squiz.cloud/squizdevreport
- https://sug-web.matrix.squiz.cloud/squizdevreport/development/carousels/_nocache

### Test Instructions

1. Load the URLs
2. Tab down the page until you reach a carousel
3. Use the keyboard to navigate through the carousel
 -- The focus (i.e. blue box) should shift to the article just panned to/selected.
4. Use the mouse to navigate a carousel
-- The focus (i.e. blue box) should not appear on the article just selected.
5. Test both keyboard and mouse navigate interchangeably
-- The focus (i.e. blue box) should only appear when using the keyboard.

# Associated Issues and/or People
- JIRA ticket(s) - https://squizgroup.atlassian.net/browse/STANFORD-994 & https://squizgroup.atlassian.net/browse/STANFORD-985
- @broleomo tagging you for approval 
- cc: @dlcousins
